exporter/pulsarexporter/README.md

```diff
@@ -2,9 +2,9 @@
 
 | Status                   |                       |
 | ------------------------ |-----------------------|
-| Stability                | [in-development]      |
+| Stability                | [alpha]               |
 | Supported pipeline types | traces, logs, metrics |
-| Distributions            | none                  |
+| Distributions            | [contrib]             |
```

Comment:
The exporter is not in the contrib distro. Please add it to https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml first

Reply:
https://github.com/open-telemetry/opentelemetry-collector-releases/pull/184 PTAL